Dockerfile

```diff
@@ -379,57 +379,55 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
 # Python
```

Comment:
commit message "copied as correct"

Reply:
> I thought the current image was already running as an user `ort`. was it not the case ?
Yes, the end image was running as ort, you are correct, but all components and the build stages were all done as root.
So now, if you look under the directory /opt, all components belongs to image built user. 
Same way, now base image, not only run image is running as user ort, so any change below that layer will be bounded to user ort, if try to escape the user jail, will fail.
Same as the copies after, that always end up with root.
Unfortunately this not solve *all* permission issues, as android-sdk tries to install in sdk root, and if user is not ort or root ( like github actions user ), we will end up in the permissions issues.